arlo_server/reports.py

```diff
@@ -0,0 +1,294 @@
+import io, csv
+from typing import Dict, List
+
+from arlo_server import app
+from arlo_server.models import (
+    Election,
+    Jurisdiction,
+    Round,
+    SampledBallot,
+    SampledBallotDraw,
+    Batch,
+    Jurisdiction,
+    BallotInterpretation,
+    Interpretation,
+    Contest,
+    RoundContest,
+)
+from arlo_server.auth import with_election_access, with_jurisdiction_access
+from util.csv_download import csv_response, election_timestamp_name
+from util.isoformat import isoformat
+from util.group_by import group_by
+
+
+def pretty_affiliation(affiliation: str) -> str:
+    mapping = {
+        "DEM": "Democrat",
+        "REP": "Republican",
+        "LIB": "Libertarian",
+        "IND": "Independent",
+        "OTH": "Other",
+    }
+    return mapping.get(affiliation, "")
+
+
+def pretty_boolean(boolean: bool) -> str:
+    return "Yes" if boolean else "No"
+
+
+def pretty_targeted(is_targeted: bool) -> str:
+    return "Targeted" if is_targeted else "Opportunistic"
```

Comment:
this feels either already reusable or soon to be. I can live with not refactoring it out but just a nudge in case it's easy.

Reply:
what do you mean "refactoring it out"? like putting it in its own module?

Reply:
yeah, I mean putting it somewhere where it can be reused, but again, if you see no use case for that, leaving it as is is fine by me.